api/python/provisioner/api.py

```diff
@@ -247,6 +248,25 @@ def set_swupgrade_repo(release, source=None, dry_run=False, nowait=False):
         If not specified then a repository for a ``release`` will be removed.
         If path to an iso file is provide then it is mounted before
         installation and unmounted before removal.
+    hash_str:
```

Comment:
I leave `hash` variable in other places, but here we need something like `hash_str` to avoid codacy issue that our local `hash` variable shadows the built-in `hash` function.